.github/workflows/ci.yml

```diff
@@ -78,7 +78,7 @@ jobs:
     strategy:
       matrix:
         platform: [amd64, arm64]
-        toolchain: [llvm, clang16, gcc12]
+        toolchain: [clang16, gcc12]
```

Comment:
For some reason, I think QA was using the llvm image to do code coverage analysis. I posted a question internally to find out.

Reply:
We will need to enable compiler-rt in the clang16 image if we want to keep the existing QA functionality (code coverage, static analysis, etc.) Does this PR do that?

Reply:
Yes.